deploy/examples/common.yaml

```diff
@@ -464,6 +464,11 @@ rules:
     verbs:
       # OBC controller updates OBC and OB statuses
       - update
+  - apiGroups: ["objectbucket.io"]
+    resources: ["objectbucketclaims/finalizers", "objectbuckets/finalizers"]
```

Comment:
How about if you add these two resources to line 463 above instead of creating a new api group?

Reply:
IMO, even thought it's more lines of config, I think it reads better to have these be separate sections. That way, finalizers are independent of status if anything needs to change in the future. Generally speaking, I think this will help keep the permissions more minimal in the long-term.

Reply:
Same thoughts for me as @BlaineEXE, but I can change it however you decide it's better. Let me know if you still think it's better to recombine the two api groups.

Reply:
It's functionally the same, so I'm fine going with this for now.